cmd/syncthing/main.go

```diff
@@ -244,6 +245,7 @@ func parseCommandLineOptions() RuntimeOptions {
 	flag.BoolVar(&options.unpaused, "unpaused", false, "Start with all devices and folders unpaused")
 	flag.StringVar(&options.logFile, "logfile", options.logFile, "Log file name (still always logs to stdout). Cannot be used together with -no-restart/STNORESTART environment variable.")
 	flag.StringVar(&options.auditFile, "auditfile", options.auditFile, "Specify audit file (use \"-\" for stdout, \"--\" for stderr)")
+	flag.BoolVar(&options.allowNewConfig, "allow-new-config", false, "Allow loading newer version of config file")
```

Comment:
Perhaps we should drop the word "file", as I don't think we refer to it being a file explicitly anywhere else. 
For all that we know the config might become a directory.